README.md

```diff
@@ -62,7 +62,7 @@ parameters:
 
 ### Setup Google oAuth login
 1. Create a Google application with your Google account by following this link: https://console.developers.google.com/
-2. Fill in needed informations. Put your Badger index page (eg. `http://badger.example.com/`) as a **valid redirect domain**
+2. Fill in needed informations. Put your Badger Google Login page (eg. `http://badger.example.com/login/check-google`) as a **valid redirect domain**
```

Comment:
Are you sure about this?
For my local development login, I use `http://localhost:8000/` as a valid redirect domain. Anyway if it fixed your problem, I see no objection to put this in the Readme :)